src/main/java/io/kestra/plugin/ldap/Search.java

```diff
@@ -120,7 +120,6 @@ public class Search extends LdapConnection implements RunnableTask<Search.Output
             SUB -- Indicates that the base entry itself and any subordinate entries (to any depth) should be considered.
             SUBORDINATE_SUBTREE -- Indicates that any subordinate entries (to any depth) below the entry specified by the base DN should be considered, but the base entry itself should not be considered, as described in draft-sermersheim-ldap-subordinate-scope."""
     )
-    @PluginProperty(dynamic = true)
```

Comment:
Sorry, I forgot to mention but here also you should only remove the attribute not the annotation.

Reply:
Done